code/modules/mob/living/carbon/superheroes.dm

```diff
@@ -0,0 +1,181 @@
+			usr << "<span class='warning'>You can only recruit Civilians.</span>"
+		if(recruiting)
+			usr << "<span class='danger'>You are already recruiting!</span>"
+			charge_counter = charge_max
+			return
+		recruiting = 1
+		usr << "<span class='danger'>This target is valid. You begin the recruiting process.</span>"
+		target << "<span class='userdanger'>[usr] focuses in concentration. Your head begins to ache.</span>"
+
+		for(var/progress = 0, progress <= 3, progress++)
+			switch(progress)
+				if(1)
+					usr << "<span class='notice'>You begin by introducing yourself and explaining what you're about.</span>"
+					usr.visible_message("<span class='danger'>[usr]'s introduces himself and explains his plans.</span>")
+				if(2)
+					usr << "<span class='notice'>You begin the recruitment of [target].</span>"
+					usr.visible_message("<span class='danger'>[usr] leans over towards [target], whispering excitedly as he gives a speech.</span>")
+					target << "<span class='danger'>You feel yourself agreeing with [usr], and a surge of loyalty begins building.</span>"
+					target.Weaken(12)
+					sleep(20)
+					if(isloyal(target))
+						usr << "<span class='notice'>They are enslaved by Nanotrasen. You feel their interest in your cause wane and disappear.</span>"
+						usr.visible_message("<span class='danger'>[usr] stops talking for a moment, then moves back away from [target].</span>")
+						target << "<span class='danger'>Your loyalty implant activates and a sharp pain reminds you of your loyalties to Nanotrasen.</span>"
+						return
+				if(3)
+					usr << "<span class='notice'>You begin filling out the application form with [target].</span>"
+					usr.visible_message("<span class='danger'>[usr] pulls out a pen and paper and begins filling an application form with [target].</span>")
+					target << "<span class='danger'>You are being convinced by [usr] to fill out an application form to become a henchman.</span>" //Ow the edge
+			if(!do_mob(usr, target, 100)) //around 30 seconds total for enthralling, 45 for someone with a loyalty implant
+				usr << "<span class='danger'>The enrollment process has been interrupted - you have lost the attention of [target].</span>"
+				target << "<span class='warning'>You move away and are no longer under the charm of [usr]. The application form is null and void.</span>"
+				recruiting = 0
+				return
+
+		recruiting = 0
+		usr << "<span class='notice'>You have recruited <b>[target]</b> as your henchman!</span>"
+		target << "<span class='deadsay'><b>You have decided to enroll as a henchman for [usr]. You are now part of the feared 'Greyshirts'.</b></span>"
+		target << "<span class='deadsay'><b>You must follow the orders of [usr], and help him succeed in his dastardly schemes.</span>"
+		target << "<span class='deadsay'>You may not harm other Greyshirt or [usr]. However, you do not need to obey other Greyshirts.</span>"
+		target.adjustOxyLoss(-200) //In case the shadowling was choking them out
```

Comment:
I don't know how you could convince somebody to fill out a form while choking them out.